src/simulator/src/plotArea.js

```diff
@@ -4,7 +4,7 @@ import { convertors } from './utils'
 var DPR = window.devicePixelRatio || 1
 
 // Helper function to scale to display
-function sh(x) {
+export function sh(x) {
```

Comment:
type the variable

Reply:
As the file is in .js format, converting it to .ts would require significant changes in both `plotArea.js` and `TimingDiagramPanel.vue`. Can we skip this for now and address it later during the TypeScript conversion phase